src/Layer/LayerUpdateStrategy.js

```diff
@@ -11,8 +11,9 @@ export const STRATEGY_GROUP = 1;
 export const STRATEGY_PROGRESSIVE = 2;
 export const STRATEGY_DICHOTOMY = 3;
 
-function _minimizeNetworkTraffic(node, nodeLevel, currentLevel) {
-    if (node.pendingSubdivision) {
+function _minimizeNetworkTraffic(node, nodeLevel, currentLevel, source) {
+    // TO DO source.isVectorTileSource is a temp fix for pendingSubdivision
```

Comment:
Can you explain why you need this fix?

Reply:
With this PR I noticed a bug with the refreshing of VectorTileLayer. The modification done on this PR makes that bug much more visible.
As it is not really the purpose of that PR to correct that bug, I just added this fix, that seems to solve it but probably not in the best way.

Reply:
issue opened #2214